source/IL2CPU.Debug.Symbols/SqliteBulkCopy.cs

```diff
@@ -50,17 +50,27 @@ public void WriteToServer(IDataReader reader)
 
                 mCommand.CommandText = $"insert into [{DestinationTableName}] ({mFieldNames.ToString().TrimEnd(',')}) values ({mParamNames.ToString().TrimEnd(',')})";
                 mCommand.Prepare();
-
+                
                 while (reader.Read())
                 {
+                    bool skipRow = false;
+                    
                     for (int i = 0; i < reader.FieldCount; i++)
                     {
                         if (parms[i] != null)
                         {
-                            parms[i].Value = reader.GetValue(i);
+                            var v = reader.GetValue(i);
+
+                            if(v == null) {
+                                Console.WriteLine("WARNING: Tried to insert null value");
```

Comment:
Do we need the Console.WriteLine here? 

We should only keep it if this fix is not final and the actual cause of the issue must be resolved at a later time.

Reply:
Well I can remove it because I believe this fix will be permanent as the affected area is only for debugging I think and I highly doubt that the non-existant argument is actually required in the debug data